src/main/java/persistence/entity/SimpleEntityManger.java

```diff
@@ -0,0 +1,59 @@
+package persistence.entity;
+
+import database.Database;
+import persistence.sql.dml.DMLQueryBuilder;
+import persistence.sql.model.PKColumn;
+import persistence.sql.model.Table;
+
+public class SimpleEntityManger implements EntityManager {
+
+    private final Database database;
+
+    public SimpleEntityManger(Database database) {
+        this.database = database;
+    }
+
+    @Override
+    public <T> T find(Class<T> clazz, Object id) {
+        DMLQueryBuilder dmlQueryBuilder = getDmlQueryBuilder(clazz);
+
+        String findByIdQuery = dmlQueryBuilder.buildFindByIdQuery(id);
+
+        return database.executeQueryForObject(clazz, findByIdQuery);
+    }
+
+    @Override
+    public void persist(Object entity) {
+        Class<?> clazz = entity.getClass();
+        DMLQueryBuilder dmlQueryBuilder = getDmlQueryBuilder(clazz);
```

Comment:
```suggestion
        DMLQueryBuilder dmlQueryBuilder = getDmlQueryBuilder(entity.getClass());
```
사소하지만 개인적으로 한번만 쓰이고, 별로 복잡하지 않다면 변수 할당하지 않는게 더 깔끔하다 생각하는데 기연님은 어떻게 생각하시나요? 😃

Reply:
저는 한 라인에서 두 가지 이상의 메서드가 호출되면 코드의 가독성을 떨어트린다고 생각되어 현재 방식으로 작성하고 있습니다!
하지만 종민님이 보여주신 코드처럼 변수할당하지 않는 코드도 깔끔해 보이네요!